content/faq/editing/advanced-editing/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: How can I do more advanced editing in OpenStreetMap?
+description: "Tutorial for editing OpenStreetMap with more advanced tools like ID"
+
+taxonomies:
+  faq: ["Editing"]
+
+extra:
+  order: 40
+---
+
+[OpenStreetMap.org](https://www.openstreetmap.org) (OSM) is a database with map data for the entire world. Similar to Wikipedia the data can be used free of charge and is crowd-sourced by millions of volunteers.
+**Organic Maps** uses the OpenStreetMap.org data to generate beautiful and detailed maps for you. 
+
+## Found wrong or missing data in the map?
+
+Did you spot a shop that doesn't exist? Or is your favorite restaurant not on the map? No problem, you can easily fix that! All map data used in Organic Maps comes from OpenStreetMap.org (OSM). And, as OSM is crowd-sourced, anyone, including you, can contribute and fix errors or add missing data. 
+
+It's best to directly fix the problem yourself as described in the [Editing OpenStreetMap](#editing-openstreetmap) section. But if you don't have time for that you can just leave an [OSM Note](#osm-notes) for other contributors.
+
+## Editing OpenStreetMap
+
+You want to add something that's missing on the map or want to fix an error? Join the community of volunteers that make OpenStreetMap possible. For beginners, it's best to use the [ID Editor](#id-editor). If you don't have a computer with internet at hand, you can also use the basic Editor of Organic Maps that works offline.
+
+### ID Editor
+
+ID is an easy-to-use, beginner-friendly editor for OpenStreetMap data. It runs in your browser, so no installation is required. You, however, need a computer, as the program is not optimized for smartphone screens. If you are new to OSM this is the editor to use.
```

Comment:
Let's highlight that ID is for desktops only, and recommend other options for mobile users (they should go before details about how to use ID on the desktop.